sass/DynamicFacet/_DynamicFacetBreadcrumbs.scss

```diff
@@ -13,3 +13,13 @@
 .coveo-dynamic-facet-breadcrumb-value-clear {
   @include breadcrumb-value-clear();
 }
+
+ul.coveo-dynamic-facet-breadcrumb.coveo-breadcrumb-item {
```

Comment:
is putting the ul/li on the css a bit too specific or unnecessary?

Reply:
That's my attemp at not trying to break any custom breadcrumb out there, that would not be using ul or li